src/main.js

```diff
@@ -57,12 +57,13 @@ var load = base.loader({
   },
 
   monitor: {
-    requires: ['profile', 'cfg'],
-    setup: ({profile, cfg}) => base.monitor({
+    requires: ['process', 'profile', 'cfg'],
+    setup: ({process, profile, cfg}) => base.monitor({
       project: 'taskcluster-index',
       credentials: cfg.taskcluster.credentials,
       mock: profile === 'test',
-    })
+      process,
+    }),
```

Comment:
he he, now we're full circle back to doing something similar to what taskcluster-lib-stats did before..
Anyways, this seems totally solid to me...